browser-test/src/applicant/navigation/application_navigation_with_eligibility_conditions.test.ts

```diff
@@ -468,27 +474,6 @@ test.describe('Applicant navigation flow', () => {
           )
         })
       })
-
-      test('shows not eligible alert on review page with ineligible answer', async ({
```

Comment:
Note to reviewers: Since this test is about the preview page, it is also not needed

Reply:
See other comment